quickshell/Modules/DankBar/Widgets/NotificationCenterButton.qml

```diff
@@ -33,4 +33,13 @@ BasePill {
             }
         }
     }
+
+    MouseArea {
```

Comment:
BasePill provides a signal, onRightClicked, it would be better to just connect to that instead of having a separate mouse handler.